libs/backend/src/cast_vote_records/export.ts

```diff
@@ -174,6 +175,7 @@ function buildCastVoteRecordReportMetadata(
     batchInfo: batches,
     election,
     electionId,
+    generatedDate: options.hideTime ? new Date(election.date) : undefined,
```

Comment:
Why is this important for voter privacy?

Reply:
And why is including it at the end of the day _not_ a problem for voter privacy?

Reply:
So we use `buildCastVoteRecordReportMetadata` to generate the metadata in two places: 1) in the top-level `metadata.json` file but also 2) in individual `cast-vote-record-report.json` files, which we now create per CVR.

If we had real date times in 2, our shuffling would become irrelevant since you could just look at the `GeneratedDate` field in the `cast-vote-record-report.json` files.

There's no issue with a real date time in 1 since it's top-level and not tied to any individual CVR.

Let me know if I can clarify further!

Reply:
Updated the comment to make this clearer in https://github.com/votingworks/vxsuite/pull/3972/commits/bc83adca281107c4fd81c5408af0802da159ba38